scss/_defs.scss

```diff
@@ -63,4 +63,16 @@ $coordinate_values: (
 );
 
 // aspect ratio
-$aspect_ratio: ();
\ No newline at end of file
+$aspect_ratio: ();
+
+// containers
+$containers: (
+        size:(
+                "container-small": 880,
+                "container": 1076,
+                "container-large": 1196,
+                "container-fluid": unset,
```

Comment:
Put a comment for `"container-fluid"`, this name is being treated differently compared to the others.